interfaces/builtin/unity8.go

```diff
@@ -0,0 +1,204 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016-2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var unity8ConnectedPlugAppArmor = []byte(`
+# Description: Can access unity8 desktop services
+# Usage: common
+
+  #include <abstractions/dbus-session-strict>
+  #include <abstractions/fonts>
+
+  # qmlscene (a common client) wants this
+  network netlink raw,
+
+  # Accessibility
+  #include <abstractions/dbus-accessibility-strict>
+  dbus (send)
+       bus=session
+       path=/org/a11y/bus
+       interface=org.a11y.Bus
+       member=GetAddress
+       peer=(name=org.a11y.Bus,label=unconfined),
+  dbus (receive, send)
+       bus=accessibility
+       path=/org/a11y/atspi/**
+       peer=(label=unconfined),
+
+  # OSK (on screen keyboard)
+  dbus (send)
+       bus=session
+       path=/org/maliit/server/address
+       interface=org.freedesktop.DBus.Properties
+       member=Get
+       peer=(name=org.maliit.server,label=unconfined),
+  unix (connect, receive, send)
+       type=stream
+       peer=(addr=@/tmp/maliit-server/dbus-*),
+
+  # Sensors
+  dbus (receive, send)
+       bus=session
+       path=/com/canonical/usensord/haptic
+       peer=(name=com.canonical.usensord,label=unconfined),
```

Comment:
Looking at the snap, we don't include usensord yet.  I'll just remove this bit until we add it back (who knows, maybe it will end up in a different snap or want its own interface...)